tests/nemo_text_processing/hy/data_inverse_text_normalization/test_cases_decimal.txt

```diff
@@ -0,0 +1,11 @@
+զրո ամբողջ մեկ~0.1
+զրո ամբողջ յոթ~0.7
```

Comment:
Here the context is also not provided to all datapoints

Reply:
it's fine if not all examples contain context, a subset of examples with context would suffice.

Reply:
I've added some examples with context. Thank you!